knimin/lib/data_access.py

```diff
@@ -2399,6 +2399,178 @@ def getProjectNames(self):
         sql = """SELECT project FROM project"""
         return [x[0] for x in self._con.execute_fetchall(sql)]
 
+    def create_sample(self, sample_ids):
+        """Creates samples by ID
+
+        Parameters
+        ----------
+        sample_ids : list of str
+            Sample IDs to create
+
+        Returns
+        -------
+        bool
+            True if successful
+        """
+        sql = """INSERT INTO pm.sample (sample_id) VALUES (%s)"""
+        self._con.executemany(sql, [[x] for x in sample_ids])
+        return True
+
+    def delete_sample(self, sample_ids):
```

Comment:
If the sample is associated with a project and/or with a sample plate, the DB will raise an error. We should probably put some checks in place before blindly removing the sample.

Also, it looks like this function is used to delete multiple samples, so it is better to update the name accordingly: `delete_samples`